modules/varbase_features/varbase_auth/varbase_auth.module

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * @file
+ * Contains varbase_auth.module.
+ */
+
+use Drupal\block\Entity\Block;
+
+/**
+ * Implements hook_page_attachments().
+ */
+function varbase_auth_page_attachments(array &$page) {
+  // Attach the styling library.
```

Comment:
Please, attache only when the URL is on the login page.
Drupal 8 machine name for the user/login page
Are the style concerned with all betterlogin TWIG files?
We do have a library for betterlogin.